### ISSUE TITLE
Accessibility options toggle lock, negative values, and combos.

### ISSUE THREAD

**Issue by LazyBunny** [matzman666/OpenVR-AdvancedSettings#171]
I need to use the accessibility options to play skyrim vr seated, as i'm unable to dive while seated as it requires me to physically move my head below my neck.

To fix this, I THINK i'd need to do a few things, I'd need the accessibility to be able to have a negative value, to lower the height below my current head height. So long as skyrim think's i've lowered my head it should work.

And i'd need to be able to toggle lock it. Right now the press to toggle, isn't a toggle. it's just a hold to activate, totally different. Toggle means click on, click off, like a light switch. If it only works while held down then it isn't toggled.

And lastly, a combo. Left Thumb Pad + Left Grip... or a keyboard key and i can bind it to Voice Attack. Or even "hold Left grip for 5 seconds to toggle on, hold for 5 to toggle off.
I can't stand for very long so seated is my only option, and taking off the headset everytime i want to dive, by moving the headset below neck height, is beyond tedious.
